src/toast/messages.tsx

```diff
@@ -0,0 +1,62 @@
+import { Toast } from "primereact/toast";
+
+export const showAdded = (toast: React.RefObject<Toast>): void => {
```

Comment:
too much same code..
create method calls
 toast.current?.show({
    severity: "...",
    summary: "Info",
    detail: "...",
    life: 3000,
  }); and reuse it in all of these methods.

Also best approach would be to have showInfo, showWarning, showSuccess like methods that just accept detail string as param. Would be much easier to call them, more readable, etc.

Reply:
Done.